sw/device/silicon_creator/manuf/base/provisioning_inputs.bzl

```diff
@@ -90,6 +90,8 @@ EARLGREY_SKUS = {
 def disqualified_for_signing(name, data):
     if "staging" in name:
         return True
+    if "emulation" in name:
+        return True
```

Comment:
LGTM, but perhaps we should add a bool flag to the config that encodes whether or not the config wants offline signing?

Reply:
Yes, I think we should. As discussed offline, we can design a flag to do this and implement in a follow up.